### PR TITLE
Supported offline 

### PR DESCRIPTION
**Pull request by ipragmatechadmin**
### Description
Added the support for offline so that app works even the internet is not available. Also fixes little issues.

### CODE REVIEW

**Comment by Qolzam** [comment]
I appreciate your pull request @kapilip . Everything work well. I haven't focused on security issues for this project much, however there is a point that redux offline save redux store data in local storage  which cause [security issues](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/HTML5_Security_Cheat_Sheet.md#local-storage). Do you have any idea about that?

**Comment by ipragmatechadmin** [comment]
@Qolzam Yes, I have read the security issues with local storage and we shouldn't be storing the sensitive information like the user's credentials or payment details in it. So IMHO we aren't storing any of these in the local storage and storing the public information which shall be available to other users. Let me know your thoughts. 
BTW you are doing a wonderful job, we have used this structure for our internal app and later added offline support for our app. We would happy to contribute to react-social-network as we add more things in our app :)

**Comment by Qolzam** [comment]
@kapilip I am happy to hear that It's helpful to your project. And I do appreciate your contribution. You are right, not sure about [next:branch](https://github.com/red-gold/react-social-network/tree/next) and if personal data which comes to redux store is important for user or not, however we need be aware of that. How is your idea that we make a pluggable feature of offline-support? Kind of option for developer!
 
 By the way I am currently working on [v0.7.0:branch **NOT READY**](https://github.com/red-gold/react-social-network/tree/v0.7.0) (I will release backend as soon as possible) in my spare time and it's going so slow, but I am trying to make this social more independent to Firebase (#109). 
Indeed I am trying to make this project closer to production. But as you know there are 531,664 lines of code (in src folder) need to be maintained and developed.
Maybe you are more interested in social network with K8S for real world production which will come after (#109). 

Thanks again!

File: src/config/environment.dev.ts [hunk 1]
```diff
@@ -2,15 +2,16 @@ import { LanguageType } from 'store/reducers/locale/langugeType'
 
 export const environment = {
   firebase: {
```
Comment (Qolzam) [inline src/config/environment.dev.ts]: Can we have the test config back for other developers to test?

Comment (ipragmatechadmin) [reply src/config/environment.dev.ts]: I have reverted back the settings

**Comment by ipragmatechadmin** [comment]
> @kapilip I am happy to hear that It's helpful to your project. And I do appreciate your contribution. You are right, not sure about [next:branch](https://github.com/red-gold/react-social-network/tree/next) and if personal data which comes to redux store is important for user or not, however we need be aware of that. How is your idea that we make a pluggable feature of offline-support? Kind of option for developer!
> 
> By the way I am currently working on [v0.7.0:branch **NOT READY**](https://github.com/red-gold/react-social-network/tree/v0.7.0) (I will release backend as soon as possible) in my spare time and it's going so slow, but I am trying to make this social more independent to Firebase (#109).
> Indeed I am trying to make this project closer to production. But as you know there are 531,664 lines of code (in src folder) need to be maintained and developed.
> Maybe you are more interested in social network with K8S for real-world production which will come after (#109).
> 
> Thanks again!

That's a good idea to make offline support as optional. Could you suggest how can I make it a plugin?

Ok, I shall try to contribute to v0.7.0 branch.  Could you tell which features you would like me to work on? I would be happy to contribute it to the project.

**Comment by ipragmatechadmin** [comment]
@Qolzam I tried to run v.0.7.0 and looks like I would also need to the source code for the API. Could you also push the code in the git repository so that I can run v0.7.0 locally?

**Comment by Qolzam** [comment]
@kapilip Thanks for checking, you are great 👍 . Can you tell why you can not start? This branch is not ready yet, It's a little bit buggy :). After completing OAuth I will push complete backed. Until next week I will organize backend too.

About plugin let's discuss here #97

**Comment by ipragmatechadmin** [comment]
@Qolzam We shall be waiting for the branch to be ready :-)

**Comment by Qolzam** [comment]
Hello @kapilip. Here is the [back-end](https://github.com/red-gold/social-firebase). Please let me know if you face with any bugs. 

Thanks!

**Comment by ipragmatechadmin** [comment]
@Qolzam Thanks a lot :-) We shall take a look and let you know if we have any question

**Pull request merged**